ephypype/fif2ts.py

```diff
@@ -19,3 +19,66 @@ def ep2ts(fif_file):
     save_path = op.abspath('ts_epochs.npy')
     save(save_path, data)
     return save_path
+
+
+def create_ts(raw_fname):
+    """Read a raw data from **.fif** file and save the data time series, the
+    sensors coordinates and labels in .npy and .txt files.
+
+    Parameters
+    ----------
+    raw_fname : str
+        pathname of the raw data to read
+
+    Returns
+    -------
+    ts_file : str
+        pathname of the numpy file (.npy) containing the data read from
+        raw_fname
+    channel_coords_file : str
+        pathname of .txt file containing the channels coordinates
+    channel_names_file : str
+        pathname of .txt file containing the channels labels
+    sfreq : float
+        sampling frequency
+    """
+    import os
+    import numpy as np
+
+    import mne
+    from mne.io import read_raw_fif
+
+    from nipype.utils.filemanip import split_filename as split_f
+
+    raw = read_raw_fif(raw_fname, preload=True)
+
+    subj_path, basename, ext = split_f(raw_fname)
+
+    select_sensors = mne.pick_types(raw.info, meg=True, ref_meg=False,
+                                    exclude='bads')
+
+    # save electrode locations
+    sens_loc = [raw.info['chs'][i]['loc'][:3] for i in select_sensors]
+    sens_loc = np.array(sens_loc)
+
+    channel_coords_file = os.path.abspath('correct_channel_coords.txt')
+    np.savetxt(channel_coords_file, sens_loc, fmt=str("%s"))
+    # np.savetxt(ROI_coords_file,np.array(ROI_coords,dtype = int),fmt = "%d")
+
+    # save electrode names
+    sens_names = np.array([raw.ch_names[pos] for pos in select_sensors],
+                          dtype='str')
+
+    channel_names_file = os.path.abspath('correct_channel_names.txt')
+    np.savetxt(channel_names_file, sens_names, fmt=str('%s'))
+
+    data, times = raw[select_sensors, :]
+
+    print((data.shape))
```

Comment:
you should have at least a `verbose` parameter. And the `print` statement should activate only if `verbose` is `True` ... 

The other way to do it is using a `logger` but I would say it's not high priority and would take time to configure.

Reply:
> you should have at least a `verbose` parameter. And the `print` statement should activate only if `verbose` is `True` ...

ok, so I can add a verbose parameter to each function where I want to print something and set by default False

Reply:
yes, the default would be `True` I think. But do it in a separate PR I would say.